packages/strapi-design-system/src/Select/SingleSelect.tsx

```diff
@@ -0,0 +1,330 @@
+import * as React from 'react';
+import * as RadixSelect from '@radix-ui/react-select';
+import { CarretDown, Cross } from '@strapi/icons';
+import styled from 'styled-components';
+
+import { Field, FieldError, FieldHint, FieldLabel } from '../Field';
+import { Stack } from '../Stack';
+import { Box } from '../Box';
+import { Flex } from '../Flex';
+import { Typography } from '../Typography';
+
+import { useId } from '../helpers/useId';
+
+import { useIntersection } from '../hooks/useIntersection';
+
+import { getThemeSize, inputFocusStyle } from '../themes/utils';
+
+export interface SingleSelectProps {
+  children: React.ReactNode;
+  /**
+   * @default "Clear"
+   */
+  clearLabel?: string;
```

Comment:
I think you didn't change that but we should make it mandatory to overwrite this because otherwise the component won't be fully translateable.

Reply:
Isn't that a user of the library problem? 🤔 I'd rather give sensible defaults and make the component easier to integrate imo – but then yet again im probably biased because the default is always english 😅

Reply:
Ok, leave it then for now. Maybe it would be good to check and unify that behavior. I think if we want to push for not forget to translate it would be beneficial to make it mandatory.

Reply:
I'll make a GH issue for it as a `discussion`.